src/test/java/org/osc/ism/OSGiIntegrationTest.java

```diff
@@ -178,46 +184,39 @@ public ApplianceSoftwareVersionElement getApplianceSoftwareVersion() {
     @Before
     public void start() throws Exception {
 
-        VirtualSystemElement vse = new VSElement(1L, VSS_TEST);
+        VSElement vs = new VSElement(1L, VSS_TEST);
+        VirtualSystemElement vse = vs;
+        String deviceId;
 
         ManagerDeviceApi managerDeviceApi = this.api.createManagerDeviceApi(null, vse);
 
-        assertNotNull(managerDeviceApi.createVSSDevice());
+        deviceId = managerDeviceApi.createVSSDevice();
+
+        assertNotNull(deviceId);
+
+        vs.setMgrId(Long.parseLong(deviceId));
 
         assertEquals(1, managerDeviceApi.listDevices().size());
 
         assertNotNull(managerDeviceApi.createDeviceMember(DEVICE_MEMBER_TEST, null, null, null, null, null));
 
+        ManagerDeviceMemberElement member = managerDeviceApi.findDeviceMemberByName(DEVICE_MEMBER_TEST);
+
+        managerDeviceApi.deleteDeviceMember(member.getId());
+
+        managerDeviceApi.deleteVSSDevice();
+
+        assertEquals(Collections.emptyList(), managerDeviceApi.listDevices());
     }
 
 
     @After
     public void stop() throws Exception {
-
-        VirtualSystemElement vse = new VSElement(1L, VSS_TEST);
-        ManagerDeviceApi managerDeviceApi = this.api.createManagerDeviceApi(null, vse);
-        ManagerDeviceMemberElement member = managerDeviceApi.findDeviceMemberByName(DEVICE_MEMBER_TEST);
-        managerDeviceApi.deleteDeviceMember(member.getId());
-        managerDeviceApi.deleteVSSDevice();
-        assertEquals(Collections.emptyList(), managerDeviceApi.listDevices());
+        //TODO - Tests needs to be added - Sudhir
```

Comment:
nit pick comment should be TODO: OWNER. We use the same format everywhere so we can track owners easily

Reply:
OK